### PR TITLE
5 ➡️ 6

### PR DESCRIPTION
**Pull request by mjcarroll**
# ➡️ Forward port

Port gz-common5 to main

Branch comparison: https://github.com/gazebosim/gz-common/compare/main...gz-common5

**Note to maintainers**: Remember to **Merge** with commit (not squash-merge or rebase)

### CODE REVIEW

File: graphics/src/AssimpLoader.cc [hunk 1]
```diff
@@ -392,8 +392,8 @@ MaterialPtr AssimpLoader::Implementation::CreateMaterial(
     auto [texName, texData] = this->LoadTexture(_scene, texturePath,
         this->GenerateTextureName(_scene, assimpMat, "MetallicRoughness"));
     // Load it into a common::Image then split it
-    auto texImg =
-      texData != nullptr ? texData : std::make_shared<common::Image>(texName);
+    auto texImg = texData != nullptr ? texData :
+      std::make_shared<common::Image>(joinPaths(_path, texName));
```
Comment (nkoenig) [inline graphics/src/AssimpLoader.cc:396]: Can you remove the `common::` namespace here?

**Review by nkoenig — COMMENTED** [review]
[test/static_assertions/CMakeLists.txt](https://github.com/gazebosim/gz-common/pull/459/files#diff-3f6df07644274a0eb71efbc8be913eba95fc4bf507e73d838cf17544db0ad364) is listed as empty. Should it be deleted?

**Comment by mjcarroll** [comment]
> [test/static_assertions/CMakeLists.txt](https://github.com/gazebosim/gz-common/pull/459/files#diff-3f6df07644274a0eb71efbc8be913eba95fc4bf507e73d838cf17544db0ad364) is listed as empty. Should it be deleted?

It should be removed now.

File: graphics/src/AssimpLoader.cc (hunk 1 above)
Comment (mjcarroll) [reply graphics/src/AssimpLoader.cc]: Done.

**Review by nkoenig — APPROVED** [review]

**Pull request merged**